pennylane/ops/cv.py

```diff
@@ -532,6 +533,61 @@ class CubicPhase(CVOperation):
     grad_method = 'F'
 
 
+class Interferometer(CVOperation):
+    r"""pennylane.Interferometer(U, wires)
+    A linear interferometer transforming the bosonic operators according to
+    the unitary matrix :math:`U`.
+
+    .. note::
+
+        This operation implements a **fixed** linear interferometer given a known
+        unitary matrix.
+
+        If you instead wish to parameterize the interferometer,
+        and calculate the gradient/optimize with respect to these parameters,
+        consider instead the :func:`pennylane.template.Interferomer` template,
+        which constructs an interferomer from a combination of beamsplitters
+        and rotation gates.
+
+    **Details:**
+
+    * None (applied to the entire subsystem)
+    * Number of parameters: 1
+    * Gradient recipe: None (uses finite difference)
+    * Heisenberg representation:
+
+      .. math:: M = \begin{bmatrix}
+        1 & 0\\
+        0 & S\\
+        \end{bmatrix}
+
+    where :math:`S` is the Gaussian symplectic transformation representing the interferometer.
```

Comment:
It would make the docs more complete if there was an explanation of how S is related to U, or at least a link to the part of the docs where this is explained.